packages/marshal/src/marshal.js

```diff
@@ -34,16 +42,20 @@ const { details: X, quote: q } = assert;
 const QCLASS = '@qclass';
 export { QCLASS };
 
+/** @type {ConvertValToSlot<any>} */
 const defaultValToSlotFn = x => x;
+/** @type {ConvertSlotToVal<any>} */
 const defaultSlotToValFn = (x, _) => x;
 
 /**
  * @template Slot
- * @type {MakeMarshal<Slot>}
+ * @param {ConvertValToSlot<Slot>} [convertValToSlot]
+ * @param {ConvertSlotToVal<Slot>} [convertSlotToVal]
+ * @param {MakeMarshalOptions} [options]
  */
-export function makeMarshal(
-  convertValToSlot = defaultValToSlotFn,
-  convertSlotToVal = defaultSlotToValFn,
+export const makeMarshal = (
```

Comment:
I missed that one. Thanks!

Reply:
Not yet, but maybe sometime.

Reply:
I would love to see this done automatically!

Reply:
an attempt: https://github.com/Agoric/agoric-sdk/pull/4608